Assets/Scripts/SquarePatrol.cs

```diff
@@ -3,7 +3,6 @@
 using UnityEngine;
 
 public class Patrol : MonoBehaviour{
-    static public float sideLength = 3.0f;
```

Comment:
These changes are already there on the main branch. You already created a PR for them last week and was approved.

This PR should've been created from the main branch instead. Either that or you merge the main branch into this branch.
From this branch, run `git merge main`